std/internal/windows/advapi32.d

```diff
@@ -16,17 +16,21 @@ private import core.sys.windows.windows;
 
 pragma(lib, "advapi32.lib");
 
-immutable bool isWow64;
-
-shared static this()
+@property immutable bool isWow64()
```

Comment:
std\internal\windows\advapi32.d(19): Error: function std.internal.windows.advapi32.isWow64 without 'this' cannot be immutable